### ISSUE TITLE
Option for disabling 'Default is'

### ISSUE THREAD

**Issue by HunterKohler** [hosseinmd/prettier-plugin-jsdoc#118]
Hello, I love the project.

I was wondering what peoples' thoughts would be on having an option for disabling the `Default is` statements that are inserted.
 
 ## Reasoning

1. Only repetitive information is inserted. The source of the defaults are from the bracketed expression, not the function parameters. The bracketed parameters serve as a good solution. If one is preferred, there shouldn't be both. So a toggle may be wanted....

2. This is not syntactically strict. There is no consideration for description. If someone has already inserted something such as `default value is x`, the formatter would have no way of detecting it, and instead inserts its own statement. The bracketed method _is_ syntactically strict. 

_\<default-variable> := [\<variable-name>=\<value>]_

### Example

Current behavior

Pre-format:
```js
/**
 * Finds the sum of two numbers.
 * 
 * @param {number} x augend
 * @param {number} [y=1] addend
 * @returns 
 */
function add(x, y = 1) {
    return x + y;
}
```

Post-format:
```js
/**
 * Finds the sum of two numbers.
 *
 * @param {number} x Augend
 * @param {number} [y=1] Addend. Default is `1`
 * @returns
 */
function add(x, y = 1) {
    return x + y;
}
```

## Other Ideas

It would also be a good idea to get the data for defaults from the function parameters itself

### Example

Theorized behavior

Pre-format:
```js
/**
 * Finds the sum of two numbers.
 * 
 * @param {number} x augend
 * @param {number} y addend
 * @returns 
 */
function add(x, y = 1) {
    return x + y;
}
```

Post-format:
```js
/**
 * Finds the sum of two numbers.
 *
 * @param {number} x augend
 * @param {number} [y=1] addend
 * @returns
 */
function add(x, y = 1) {
    return x + y;
}

**Comment by hosseinmd** [comment]
Good idea, I will work on it few days later. You could also create a pull request.

**Comment by hosseinmd** [comment]
Released v0.3.29
